cssBundle/footer.css

```diff
@@ -0,0 +1,18 @@
+/* .copyright-text {
+  padding: 0 var(--spacing-xs);
+} */
```

Comment:
Suggestion: Can remove commented code if not necessary

Reply:
yeah removed it in the child branch